kernel/src/syscall/fcntl.rs

```diff
@@ -164,11 +164,11 @@ enum FcntlCmd {
     F_SETFD = 2,
     F_GETFL = 3,
     F_SETFL = 4,
+    F_GETLK = 5,
     F_SETLK = 6,
     F_SETLKW = 7,
-    F_GETLK = 8,
-    F_SETOWN = 9,
-    F_GETOWN = 10,
+    F_SETOWN = 8,
+    F_GETOWN = 9,
```

Comment:
But we should have tested this fcntl sub-commands. How can the command number be incorrect?

@grief8 Any inputs?

Reply:
This PR makes sense after I checked the [code](https://elixir.bootlin.com/linux/v6.10/source/include/uapi/asm-generic/fcntl.h#L111). Previous syscall tests can pass the tests without the cases about `F_GETLK`.

Reply:
@sdww0 Hi, BTW, could you also remove the block lists about `F_SETOWN` under `asterinas/test/syscall_test/blocklists/fcntl_test` ?